alpha/alpha-fsm-channel-kafka/README.md

```diff
@@ -0,0 +1,17 @@
+# FSM kafka channel
+## Enabled Saga State Machine Module
+
+Using `alpha.feature.akka.enabled=true` launch Alpha and Omega Side 
+Using `alpha.feature.akka.channel.type=kafka` launch Alpha and Omega Side 
+
+```properties
+alpha.feature.akka.enabled=true
+alpha.feature.akka.channel.type=kafka
+```
+
+setting spring boot kafka
+```
+spring.kafka.bootstrap-servers=kafka bootstrap_servers 
+spring.kafka.consumer.group-id=kafka consumer group id
+alpha.feature.akka.channel.kafka.topic= kafka topic name
```

Comment:
I recommend specifying the parameter `alpha.feature.akka.channel.kafka.topic` defaults in the documentation and not recommending user modifications.

Reply:
add doc to readme